app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesFragment.java

```diff
@@ -178,6 +178,18 @@ public void onChanged() {
             }
         };
         stickyHeaderAdapter.registerAdapterDataObserver(adapterDataObserver);
+
+        recyclerView.addOnScrollListener(new RecyclerView.OnScrollListener() {
```

Comment:
> remove listener in onDestroy

Actually I left it intentionally. I didn't understand the need of removing listener in `onDestroy`, since anyway after `onDestroy` recyclerView will be destroyed. Hence no need to save the listener separately and destroy it in `onDestroy` call. Better pass it directly to the recyclerView.

what do you say?